book/src/users/README.md

```diff
@@ -39,18 +49,17 @@ network with additional robustness.
 ## Mobile user
 
 Trin is not currently configured to run on mobile, however this is plausibly
-a viable and interesting use case. A trin node could run as a background
-task with configurable limits on disk, CPU and bandwidth use.
+a viable and interesting use case for the future. There are a number of
+challenges to address first. Mobile does not typically support backrgound use
+of apps, and the battery life of a mobile device is a concern. So one challenge
+is how to make the mobile clients contribute back to the network in a way that
+is not too burdensome on the device.
 
-*Benefit*: Wallet use without reliance on third party wallet APIs. Contributes to
-network health.
+*Benefit*: Wallet use without reliance on third party wallet APIs.
 
 ## Unsuitable users
 
 There are situations where Trin is estimated to not be a good node choice:
-- Time-critical chain tip data. Likely that data distribution may not be fast enough for these
-    use cases, however testing may show otherwise.
-    - Consensus participation. Beacon chain staking with a Consensus client with Portal Network node as Execution client.
-    - Block builder. Serving blocks to beacon chain validator nodes via MEV-boost
-- Data analysis requiring state at historical blocks. Trin is not an archive node and does not
-    expose `trace_`* or` debug_`* endpoints.
+- Very speedy historical state access. It's possible to retrieve old state, but don't expect sub-second contract reads on state as viewed from a historical block.
+- Building blocks locally, as a block producer. Random access to the full state and transaction pool is not supported at the speed needed to build competitive blocks.
+  - We remain hopeful that in the future, you could use an externally-generated block (like one provided by MEV-boost) so that you can act as a validater using a standard Consensus client, with Trin as the Execution client. This probably depends on a future where state witnesses are bundled with the block sent to you by the producer.
```

Comment:
I think the PR is good to merge, I am just pointing out here that I would like to discuss the future of Trin Execution during Devcon as I outlined here https://github.com/ethereum/trin/pull/1556#discussion_r1821786793.

So I do disagree with `Trin as the Execution client`, but I don't think that disagreement should block this PR from being merged and we can discuss that issue at Devcon.